ml_enabler/models/ml_model.py

```diff
@@ -2,6 +2,40 @@
 from ml_enabler.models.utils import timestamp
 
 
+class Prediction(db.Model):
+    """ Predictions from a model at a given time """
+    __tablename__ = 'predictions'
+
+    id = db.Column(db.Integer, primary_key=True)
+    created = db.Column(db.DateTime, default=timestamp, nullable=False)
+    model_id = db.Column(db.BigInteger, db.ForeignKey(
+                        'ml_models.id', name='fk_models'), nullable=False)
+    predictions = db.Column(db.JSON, nullable=False)
```

Comment:
@geohacker, `predictions` will just be a JSON with a bunch of entries for tile ID, layer/image, prediction value, correct?

Reply:
Yes, that's right. The entries will be mapped to the tile id `x/y/z`: {...}